element_miniscope/miniscope.py

```diff
@@ -102,21 +77,17 @@ def get_miniscope_root_data_dir() -> list:
 
 
 def get_session_directory(session_key: dict) -> str:
-    """
-    get_session_directory(session_key: dict) -> str
-        Retrieve the session directory containing the
-         recorded data for a given session
-        :param session_key: a dictionary of one session `key`
-        :return: a string for relative or full path to the session directory
+    """Pulls relevant directory information from database.
+
+    Args:
+    session_key (dict): containing session information
```

Comment:
Args need further indenting
```suggestion
        session_key (dict): containing session information
```

Reply:
Fixed